data/hails.txt

```diff
@@ -1127,6 +1208,10 @@ phrase "friendly navy"
 		"Answer the call of duty! Join the Navy!"
 		"If you have courage, if you have honor, join the Navy!"
 		"We want YOU for the Navy!"
+		"Defend the galaxy. Join the Navy!"
+		"All citizens are subject to random cargo scans. Comply or face criminal charges."
+		"Cargo scans will be conducted at random intervals by all Navy vessels. Comply or face criminal charges."
+		"You are subject to cargo scans. Comply or face the charge of treason."
```

Comment:
Hah, I like these!

Reply:
Well, uh, these last three are amusing, but they don't really fit with the Navy's positive theme.

Reply:
That's true, the tone could use some work, but I do like the idea of mentioning all the scans.

Reply:
Blade, it is a criminal act to force them to not scan you no? You'd have to enter combat to make them stop. afaik, and if you get those smuggling plates I'd assume that'd be considered an illegal mod.

Reply:
Interference Plating isn't marked as illegal, so no.

Reply:
Obviously, cause that'd cause a problem with the item itself. It's something that can't be detected so easily as it's part of the ship's walls. Doesn't mean it's not illegal.

Reply:
That's not the point. The other ones here praise the Navy and the Republic or a united humanity. These scanning statements make pretty hostile demands.

Reply:
the concept is nice, just make it "less agressive"